test/SIL.XForge.Scripture.Tests/Services/ParatextSyncRunnerTests.cs

```diff
@@ -2882,15 +2882,6 @@ public void SetupNoteChanges(string threadId, string verseRef = "MAT 1:1", bool
                         )
                         .Returns(userIdsToUsernames);
                 }
-                else
```

Comment:
I'm missing the context for why this was removed.

Reply:
Nothing was returned from the code. Variables were set, but they were not returned.

Reply:
Ah, right. There were no side effects.

Reply:
Ah, right. There were no side effects.